test/util.test.ts

```diff
@@ -0,0 +1,54 @@
+import { promises } from 'fs';
+import { tmpdir } from 'os';
+import path from 'path';
+import { getFiles } from '../src/util';
+
+describe('getFiles', () => {
+
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('test read files', async () => {
+
+    //WHEN we have this folder structure
+
+    // Only look for files of this type
+    const extensionFilter = 'k8s.yaml';
+    // Create folders
+    const baseDir: string = await promises.mkdtemp(path.join(tmpdir(), 'cdk8s-getFiles-test'));
+    const emptySubDir: string = await promises.mkdtemp(path.join(baseDir, 'emptyDir'));
+    const PopulatedSubDir: string = await promises.mkdtemp(path.join(baseDir, 'populatedDir'));
+    const PopulatedSubDir2: string = await promises.mkdtemp(path.join(PopulatedSubDir, 'populatedDir2'));
```

Comment:
We prefer to use lowerCamelCase for variables to be consistent throughout our codebase:
```suggestion
    const populatedSubDir: string = await promises.mkdtemp(path.join(baseDir, 'populatedDir'));
    const populatedSubDir2: string = await promises.mkdtemp(path.join(populatedSubDir, 'populatedDir2'));
```

Reply:
Thanks for the suggestions @Chriscbr! The changes are done and committed